src/dawn/CodeGen/CXXNaive/ASTStencilFunctionParamVisitor.h

```diff
@@ -44,7 +45,7 @@ class ASTStencilFunctionParamVisitor : public ASTVisitorDisabled, public NonCopy
   using Base = ASTVisitorDisabled;
 
   ASTStencilFunctionParamVisitor(const std::shared_ptr<iir::StencilFunctionInstantiation>& function,
-                                 iir::StencilInstantiation const* instantiation);
+                                 const iir::IIR *iir);
```

Comment:
notice you changed the semantic, before it was pointer to const object. now is const pointer to object